org.eclipse.dartboard/plugin.xml

```diff
@@ -169,4 +169,14 @@
              type="java.lang.Object">
        </propertyTester>
     </extension>
+    <extension
+          point="org.eclipse.debug.ui.launchGroups">
+       <launchGroup
+             bannerImage="platform:/plugin/org.eclipse.pde.ui/icons/obj16/run_exc.png"
```

Comment:
This is marked as required, what Image could I use here?

Reply:
Actually, we need to make icons for this product. Still, we need to get approval  ?
We can use  these temporarily
             bannerImage="platform:/plugin/org.eclipse.debug.ui/icons/full/wizban/run_wiz.png"
             image="platform:/plugin/org.eclipse.debug.ui/icons/full/etool16/run_exc.png"

Reply:
Yeah, still waiting for approval by the eclipse foundation. Should be the next couple of days. I'm going to provide a patch changing out all icons once we have full approval. 

For now I think it's fine to merge this with platform assets.